### PR TITLE
Blog post

### LINKED ISSUE #16: Blog Post page styling

### PR DESCRIPTION
**Pull request by qilto**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #16

Test URLs:
- Before: https://main--uhg-surest--hlxsites.hlx.page/blog/what-is-health-equity
- After: https://blog-post--uhg-surest--hlxsites.hlx.page/blog/what-is-health-equity

### CODE REVIEW

File: blocks/hero/hero.js [hunk 1]
```diff
@@ -12,7 +50,12 @@ export default function decorate(hero) {
   if (pic) {
     const heroMedia = createElement('div', 'hero-media-wrapper');
     heroMedia.append(pic);
-    hero.prepend(heroMedia);
+    if (document.querySelector('meta[name="template"][content="Blog Post"]')) {
```
Comment (shsteimer) [inline blocks/hero/hero.js:53]: you can just use lib-franklin getMetadata() for this?

File: blocks/hero/hero.js [hunk 2]
```diff
@@ -1,5 +1,43 @@
 import { createElement } from '../../scripts/scripts.js';
 
+function decorateBlogHeader(block) {
```
Comment (shsteimer) [inline blocks/hero/hero.js:3]: I kinda don't think I want all of this stuff mixed up in the hero. Lets chat and figure out the best way to do this

Comment (badvision) [reply blocks/hero/hero.js:3]: I agree -- we really want the blog stuff to be in its own file somehow.  Does franklin understand component-theme.css or component-template.css variations?

**Review by badvision — CHANGES_REQUESTED** [review]
Same questions/concerns as @shsteimer

File: blocks/hero/hero.js (hunk 2 above)
Comment (qilto) [reply blocks/hero/hero.js:4]: @badvision We decided to leave this with hero for now, but will probably separate it out later.

**Pull request merged**